### PR TITLE
address breaking change in dplyr::last()

### LINKED ISSUE #490: bug in adorn_percentages(denominator = "col")
``` r
      library(janitor)      
      mtcars %>%
        tabyl(am, cyl) %>%
        adorn_totals(c("row", "col")) %>%
        adorn_percentages(denominator = "col")
#> Error in FUN(X[[i]], ...): only defined on a data frame with all numeric-alike variables
```

### PR DESCRIPTION
**Pull request by sfirke**
## Description
Fix #490

### CODE REVIEW

**Comment by lionel-** [comment]
Thanks for this! We plan to release dplyr 1.1.0 on January 27. A pre-emptive fix release of janitor would be helpful.

File: R/adorn_percentages.R [hunk 1]
```diff
@@ -91,9 +91,9 @@ adorn_percentages <- function(dat, denominator = "row", na.rm = TRUE, ...) {
       if ("col" %in% attr(dat, "totals") & !explicitly_exempt_totals) {
         cols_to_tally <- c(cols_to_tally, ncol(dat))
         if ("row" %in% attr(dat, "totals")) {
-          col_sum <- c(col_sum, sum(dplyr::last(dat)[-nrow(dat)]))
+          col_sum <- c(col_sum, sum(dat[-nrow(dat), ncol(dat)]))
```
Comment (lionel-) [inline R/adorn_percentages.R:94]: Just FYI, another way to extract the last column with dplyr is `dplyr::pull(dat)`.

**Comment by sfirke** [comment]
> Thanks for this! We plan to release dplyr 1.1.0 on January 27. A pre-emptive fix release of janitor would be helpful.

That's the date for submission to CRAN?  I appreciate the advance notice, we will plan to submit the next version of janitor to CRAN prior to that.

**Comment by lionel-** [comment]
> That's the date for submission to CRAN?

yup it is

> we will plan to submit the next version of janitor to CRAN prior to that.

Thanks!

**Pull request merged**